.gitignore

```diff
@@ -81,3 +81,4 @@ prime/
 *.snap-build
 *_source.tar.bz2
 .DS_Store
+.fuse_*
```

Comment:
You can put user specific file ignores in `~/.gitignore`

Reply:
thanks - done